tests/expr_and_series/cast_test.py

```diff
@@ -236,7 +236,9 @@ def test_cast_datetime_tz_aware(
 
 
 def test_cast_struct(request: pytest.FixtureRequest, constructor: Constructor) -> None:
-    if any(backend in str(constructor) for backend in ("dask", "modin", "cudf")):
+    if any(
+        backend in str(constructor) for backend in ("dask", "modin", "cudf", "sqlframe")
+    ):
```

Comment:
Regardless having it more generic in line 256, I could not manage to make the test pass with sqlframe.
Might require investigation.

```terminal
self = <sqlframe.duckdb.session.DuckDBSession object at 0x7fcf53fa0380>
sql = 'WITH "t26313807" AS (SELECT CAST("a" AS MAP(TEXT, TEXT)) AS "a" FROM (VALUES (MAP([\'movie \', \'rating\'], [\'Cars\'...AS DOUBLE)} AS "a" FROM "t26313807") SELECT CAST("a" AS STRUCT("movie" TEXT, "rating" DOUBLE)) AS "a" FROM "t10314570"'

    def _execute(self, sql: str) -> None:
>       self._last_result = self._cur.execute(sql)  # type: ignore
E       duckdb.duckdb.BinderException: Binder Error: STRUCT to STRUCT cast must have at least one matching member

.venv/lib/python3.12/site-packages/sqlframe/duckdb/session.py:75: BinderException
```